packages/core/src/indexing-store/cache.ts

```diff
@@ -233,6 +241,57 @@ const getBytes = (value: unknown) => {
   return size;
 };
 
+export const getCopyText = (
+  table: Table,
+  rows: { [key: string]: unknown }[],
+) => {
+  const columns = Object.entries(getTableColumns(table));
+  const results: string[] = [];
+  for (const row of rows) {
+    const values: string[] = [];
+    for (const [columnName, column] of columns) {
+      let value = row[columnName];
+      if (value === null || value === undefined) {
+        values.push("\\N");
+      } else {
+        if (column.mapToDriverValue !== undefined) {
+          value = column.mapToDriverValue(value);
+        }
+        if (value === null || value === undefined) {
+          values.push("\\N");
+        } else {
+          values.push(String(value).replace(/\\/g, "\\\\"));
+        }
+      }
+    }
+    results.push(values.join("\t"));
+  }
+  return results.join("\n");
+};
```

Comment:
@typedarray Is this what you had in mind?

Reply:
Yup LGTM